src/app/Scenes/MyProfile/Components/UserAccountHeader/UserAccountHeader.tsx

```diff
@@ -0,0 +1,250 @@
+import {
+  MapPinIcon,
+  PersonIcon,
+  ShieldIcon,
+  BriefcaseIcon,
+  InstitutionIcon,
+} from "@artsy/icons/native"
+import {
+  Flex,
+  Text,
+  Touchable,
+  useScreenDimensions,
+  useSpace,
+  Image,
+  useColor,
+  Join,
+  Spacer,
+  Button,
+  Spinner,
+} from "@artsy/palette-mobile"
+import { UserAccountHeaderQuery } from "__generated__/UserAccountHeaderQuery.graphql"
+import { UserAccountHeader_me$key } from "__generated__/UserAccountHeader_me.graphql"
+import { ErrorMessageComponent } from "app/Scenes/MyProfile/Components/UserAccountHeader/ErrorMessageComponent"
+import { MyCollectionPreview } from "app/Scenes/MyProfile/Components/UserAccountHeader/MyCollectionPreview"
+import { navigate } from "app/system/navigation/navigate"
+import { withSuspense } from "app/utils/hooks/withSuspense"
+import { graphql, useFragment, useLazyLoadQuery } from "react-relay"
+
+interface UserAccountHeaderProps {
+  meProps: UserAccountHeader_me$key
+}
+export const UserAccountHeader: React.FC<UserAccountHeaderProps> = ({ meProps }) => {
+  const me = useFragment<UserAccountHeader_me$key>(userAccountHeaderFragment, meProps)
+
+  const { width } = useScreenDimensions()
+  const space = useSpace()
+  const color = useColor()
+  const WIDTH = width - 2 * space(2)
+
+  if (!me) {
+    return ErrorMessageComponent
+  }
+
+  const isProfileComplete =
+    !!me.location?.display && !!me.profession && !!me.icon?.url && !!me.isIdentityVerified
+
+  return (
+    <Flex justifyContent="center" alignItems="center" mt={2}>
+      <Touchable testID="account-card" onPress={() => navigate("/my-collection")}>
```

Comment:
https://github.com/artsy/eigen/blob/816c8171e80e71806aecf2ee097c03eaef947cc9/docs/routing.md#L1

```suggestion
      <RouterLink to="/my-collection" testID="account-card">
```